server/src/main/java/com/tencent/rss/server/LocalStorageChecker.java

```diff
@@ -118,7 +122,10 @@ boolean checkIsHealthy() {
         this.isHealthy = false;
         return false;
       }
-      double usagePercent = getUsedSpace(storageDir) * 100.0 / getTotalSpace(storageDir);
+      // We assume that this partition is only used by us so the used space is the local shuffle data size.
```

Comment:
Calculate the used space, the comment is not necessary for such simple case.

Reply:
`getUsedSpace` and `getTotalSpace` actually get the partition usage info, which is not the actual data size in the storage base path. So I add a comment here to make an explanation about it.

Reply:
comment: Get size of shuffle data, the value maybe incorrect if the folder is used by another propose.

Reply:
ok